rllib/algorithms/ppo/ppo_catalog.py

```diff
@@ -71,31 +71,22 @@ def __init__(
         post_fcnet_activation = self.model_config_dict["post_fcnet_activation"]
 
         self.pi_head_config = MLPHeadConfig(
-            input_dim=self.encoder_config.output_dim,
+            input_dims=self.latent_dims,
```

Comment:
Catalog now has latent_dims that users can set manually instead of relying on our encoder configs.